libs/facilities/frontend/view/src/lib/components/detail/detail.page.html

```diff
@@ -6,28 +6,29 @@
 	>
 	</ix-content-header>
 
-	<div class="flex flex-wrap gap-x-8">
-		<div echarts class="chart" [options]="pumpOptions" [theme]="theme"></div>
+	<div class="details-grid gap-x-6 gap-y-4 pb-12">
 
-		<div echarts class="chart" [options]="environmentOptions" [theme]="theme"></div>
+        <lib-chart [data]="pumpData" [title]="'Pump Data'" [yname]="'Flow'" [color]="'#07dbc7'"></lib-chart>
 
-		<ix-card variant="insight" class="infoContainer nochart">
+        <lib-chart [data]="envData" [title]="'Environment Data'" [yname]="'Humidity'" [color]="'#07a307'"></lib-chart>
```

Comment:
Same as above

If u want to should different graphs use @for and mock all the data in the TS array

Reply:
There are only 2 graphs to display. One for pump data and one for environment data. I think using @for for 2 entries is overkill